BSHUF/config/cmake/CTestScript.cmake

```diff
@@ -25,30 +25,23 @@ if (NOT SITE_OS_NAME)
   message (STATUS "Dashboard script uname output: ${osname}-${osrel}-${cpu}\n")
 
   set (CTEST_BUILD_NAME  "${osname}-${osrel}-${cpu}")
+  if (SITE_BUILDNAME_SUFFIX)
+    set (CTEST_BUILD_NAME  "${SITE_BUILDNAME_SUFFIX}-${CTEST_BUILD_NAME}")
+  endif ()
+  set (BUILD_OPTIONS "${ADD_BUILD_OPTIONS}")
 else ()
   ## machine name provided
   ## --------------------------
   if (CMAKE_HOST_UNIX)
-    set (CTEST_BUILD_NAME "${SITE_OS_NAME}-${SITE_OS_VERSION}-${SITE_OS_BITS}-${SITE_COMPILER_NAME}-${SITE_COMPILER_VERSION}")
+    set(CTEST_BUILD_NAME "${SITE_OS_NAME}-${SITE_OS_VERSION}-${SITE_OS_BITS}-${SITE_COMPILER_NAME}-${SITE_COMPILER_VERSION}")
```

Comment:
Why is the space between set and "(CTEST..." removed from lines 32 & 34 while new "set (CTEST..." lines in lines 31 & 43 have the space between?

Reply:
no reason